docs/docs/segment-spotify.md

```diff
@@ -6,11 +6,14 @@ sidebar_label: Spotify
 
 ## What
 
-Show the currently playing song in the Spotify MacOS client. Only available on MacOS for obvious reasons.
-Be aware this can make the prompt a tad bit slower as it needs to get a response from the Spotify player using Applescript.
+Show the currently playing song in the Spotify MacOS/Windows client.
+AutoHotkey is needed for the windows client. You also have to add the
+`autohotkey_script` property with the path to this [script](./segment-spotify.ahk) in the spotify section of your config file.
+Be aware this can make the prompt a tad bit slower as it needs to get a response from the Spotify player using Applescript/AutoHotkey.
 
 ## Sample Configuration
 
+### darwin
```

Comment:
I would keep the JSON sample the same, and only differentiate below under `Properties` eg:

```markdown
### Properties

- playing_icon: `string` - text/icon to show when playing - defaults to `\uE602 `
- paused_icon: `string` - text/icon to show when paused - defaults to `\uF8E3 `
- stopped_icon: `string` - text/icon to show when paused - defaults to `\uF8E3 `
- track_separator: `string` - text/icon to put between the artist and song name - defaults to ` - `

### Windows
- autohotkey_script: `string` - path to the autohotkey script - defaults to `""`

//TODO: add autohotkey explanation and script here
```